amnesia/src/XCompileToWindows.toolchain

```diff
@@ -1,26 +1,29 @@
-# the name of the target operating system
+# The name of the target operating system
 set(CMAKE_SYSTEM_NAME Windows)
 
-# which compilers to use for C and C++
-# set(CMAKE_C_COMPILER   x86_64-w64-mingw32-gcc-win32)  # 64 bit but the windows version was made for 32 only
-# set(CMAKE_CXX_COMPILER x86_64-w64-mingw32-g++-win32)  # 64 bit
-set(CMAKE_C_COMPILER   i686-w64-mingw32-gcc-win32)
-set(CMAKE_CXX_COMPILER i686-w64-mingw32-g++-win32)
+# Which compilers to use for C and C++
+# Uncomment these lines for 64-bit compilation, though these are set for a 32-bit Windows environment
+# set(CMAKE_C_COMPILER   x86_64-w64-mingw32-gcc-win32)  # 64-bit GCC compiler, targeting 32-bit Windows
+# set(CMAKE_CXX_COMPILER x86_64-w64-mingw32-g++-win32)  # 64-bit G++ compiler
 
-# these shouldn't be necessary, in theory...
+# Use these lines for 32-bit compilation
+set(CMAKE_C_COMPILER   i686-w64-mingw32-gcc-win32)     # 32-bit GCC compiler
+set(CMAKE_CXX_COMPILER i686-w64-mingw32-g++-win32)     # 32-bit G++ compiler
+
+# These options force 32-bit compilation, which shouldn't be necessary as the compilers specified are already for 32-bit
```

Comment:
Again, is this true? As far as I'm aware `x86_64-w64-mingw32-*-win32` are for 64 bit and `win32` is just the platform name for windows in general. There is no `mingw64` or `-win64` compilers in mingw.

Reply:
Likely due to confusion on my end.

Removed.